lib/datadog/ci/ext/environment.rb

```diff
@@ -312,6 +317,107 @@ def extract_bitrise(env)
           }
         end
 
+        def git_commit_users
+          # Get committer and author information in one command.
+          #
+          # Because we can't get a reliable UTC time from all recent versions of git
+          # We have to rely on converting the date to UTC ourselves.
```

Comment:
I added it here because it's where the git command that returns the date format is: `git show -s --format='%an\t%ae\t%at\t%cn\t%ce\t%ct'`.

But if you are confused by it, then I'll be too in a few months, so I'm moving it to the date formatting place.